drivers/serial/uart_native_posix.c

```diff
@@ -364,14 +364,14 @@ static int np_uart_tty_poll_in(const struct device *dev,
 	return 0;
 }
 
-DEVICE_AND_API_INIT(uart_native_posix0,
-	    DT_INST_LABEL(0), &np_uart_0_init,
+DEVICE_DT_INST_DEFINE(0,
+	    &np_uart_0_init, device_pm_control_nop,
 	    (void *)&native_uart_status_0, NULL,
 	    PRE_KERNEL_1, CONFIG_KERNEL_INIT_PRIORITY_DEVICE,
 	    &np_uart_driver_api_0);
 
 #if defined(CONFIG_UART_NATIVE_POSIX_PORT_1_ENABLE)
-DEVICE_AND_API_INIT(uart_native_posix1,
+DEVICE_DT_INST_DEFINE(uart_native_posix1,
```

Comment:
```suggestion
DEVICE_DT_INST_DEFINE(1,
```

Reply:
I'm actually going to leave this alone and revert it back to `DEVICE_AND_API_INIT`.  Need to cleanup the `PORT_1` posix stuff to use device tree instead (but that should be a follow up PR).